src/index.css

```diff
@@ -1,10 +1,31 @@
 :root {
     --font-stack-default: Inter, system-ui, "Segoe UI", sans-serif;
-
     color-scheme: light;
     font-family: var(--font-stack-default);
     font-size: 16px;
     line-height: 1.5;
+    --color-bg: rgb(98% 98% 98%);
+    --color-fg: rgb(12% 12% 12%);
+    --color-fg-variant: rgb(60% 60% 60%);
+    --color-border: rgb(86% 86% 86%);
+    --color-accent: rgb(39.22% 42.35% 100%);
+}
+
+html {
```

Comment:
Hm, do you prefer not having a separate `reset.css`?

I’m not sure what the convention is when working with React. My rationale behind having a separate `reset.css` file was so that we can separate the styling that’s to normalise rendering behaviours across browsers, versus styling used to implement our design. It frees `index.css` of boilerplate that’s not actually to do with design.

(Not having a separate `colours.css` I’m completely okay with, though. I see it both ways in that case.)

Reply:
Mmm I see what you mean. My thought process was that since `index.css` is applied over the whole application, similar to what `reset.css` does, it made sense to localize these stylings into one file. Also given that both of these files are updating the style of our application at the "same level" I felt like it might be hard to maintain as we might make changes to the changes to the `body` element in both files, so then how do we know which one is being applied. I feel like if we have them in 1 file it makes it easier for us to manage this and then if we wish to make individual changes to elements for each component, we can do so in each of their own CSS files.

With that being said I am open to having both `reset.css` and `index.css` , but we as a team just have to be aware of that.

Reply:
It’s certainly common practice to have a `reset.css`. Critically, it should be the **first** stylesheet that gets imported. And because subsequent styling on the same element overwrites previous ones, we don’t run into precedence issues.

(Some people call it `normalize.css`.)

This is because sometimes browsers like to set default margin/padding values that aren’t sensible (like `0`). The purpose of reset literally is just to level the playing field, so that we can much more predictably know how our tool will look in different browsers.